### DISCUSSION TITLE
How can I turn off the discovery output?

### DISCUSSION THREAD

**Discussion by claudio-salvio** [pester/Pester#2137] [Q&A] [answered]
Hello:

Currently if for example I run :
```powershell
. '.\Add-DoubleBacktickPrefixToSquareBracket.tests.ps1'
```
I get the following output:
![image](https://user-images.githubusercontent.com/66575834/153006431-bee6cccf-82b3-4f9f-8184-0d4045aaf0ec.png)

I would want to get only this:
![image](https://user-images.githubusercontent.com/66575834/153006614-31a71f50-4a31-4fd9-bce7-d3cebcc7bfca.png)

Is there any configuration to achieve this?

Greetings,
Claudio

I💙Pester

**Comment by fflaten** [comment, marked as answer]
They're not configurable without disabling all output (`-Output None`) atm.

We include more with Detailed and Diagnostic output, but those three are mandatory in the default output.

**Reply by claudio-salvio** [reply]
@fflaten,
Thank you for your reply.

It seems to me that it would be useful to **add another verbosity option**, let's call it **Reduced** that shows only 2 lines: 
the line with the name of the test file and the duration of the execution, for example:
```powershell
[+] C:\tests\FunctionA.tests.ps1 216ms (77ms|105ms)
```
and the line with the quantities of the results of the executed tests, for example:
```powershell
Tests Passed: 5, Failed: 0, Skipped: 0 NotRun: 0
```

Greetings,
Claudio

**Reply by fflaten** [reply]
You're welcome to open a feature request issue for it so we can collect feedback from others and measure interest.

Personally I don't see the need. We've already reduced the Normal output to a minimum so it is already the reduced mode.

Remember that time consuming code may also happen in the discovery-phase. By hiding those lines you would not get any indication that your discovery code (ex. generation of tests from data/input or misplaced code) are taking a very long time. 

Using `$report = Invoke-Pester -Output None -Passthru` you can suppress all default output and easily create the output you want by looking at the returned object, though first after it's done.
